### PR TITLE
Create my council account

### PR DESCRIPTION
**Pull request by ssrome**
- Added the local host as a base url in Cypress config so only have to put something like "/" or "/my-council-account/" in the test to get to page
- Tests for certain elements/text on the page
- Page with both buttons. Need to add the page name of where it should go
- Added an extra instruction to readme, for running tests - in separate PR

my-council-account

### CODE REVIEW

File: cypress.config.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ const { defineConfig } = require("cypress");
 
 module.exports = defineConfig({
   e2e: {
+    baseUrl: "http://localhost:3000",
```
Comment (Joseaie) [inline cypress.config.js:5]: Nice!
nit: But technically should be part of a separate PR. 👀

File: cypress/e2e/spec.cy.js [hunk 2]
```diff
@@ -1,19 +1,19 @@
-describe('empty spec', () => {
-  it('passes', () => {
-    cy.visit('http://localhost:3000')
-    cy.contains('GOV.UK')
-  })
-})
+describe("empty spec", () => {
```
Comment (Joseaie) [inline cypress/e2e/spec.cy.js:1]: nit: these changes should be in a separate PR

File: readme.md [hunk 3]
```diff
@@ -35,7 +35,9 @@ Use `npm run devstart` to run server locally. This is so you don't have to resta
 
 ## Run Cypress tests locally
 
-Use `npm run e2e`
+Use `npm run e2e`. This will start the server, run the tests and close the server
```
Comment (Joseaie) [inline readme.md]: nit: these changes should be in a separate PR

File: templates/default-page-template.html [hunk 4]
```diff
@@ -1,99 +1,173 @@
 <!DOCTYPE html>
-<html lang="en" class="govuk-template ">
```
Comment (Joseaie) [inline templates/default-page-template.html:2]: nit: these changes should be in a separate PR

File: public/my-council-account/index.html [hunk 5]
```diff
@@ -0,0 +1,232 @@
...
+
+          <button
+            class="govuk-button govuk-button--secondary"
+            data-module="govuk-button"
+          >
+            Log in
+          </button>
+        </div>
+
+        <!-- <div class="govuk-body govuk-button-group">
```
Comment (Joseaie) [inline public/my-council-account/index.html]: nit: do we need the commented out bits?

**Comment by Joseaie** [comment]
I've left some comments on sections that should be part of different PRs as they don't relate to the title of this one (Single Responsibility and dat (even though that refers to code)). Other than that, I think your tests look pretty good, though I'm not sure if you wanted to update them with the `data-*` attributes.

Well done ✨

**Comment by ssrome** [comment]
Gonna spilt these changes into separate PRs

**Comment by ssrome** [comment]
The other pages that are showing as changed, are formatting things that I get fix. The actual changes for those files are in the separate PRs. This one should just be my tests and my council account page

File: public/my-council-account/index.html (hunk 5 above)
Comment (ssrome) [reply public/my-council-account/index.html]: Now deleted this

**Review by LarabfSantos — APPROVED** [review]
